components/site/src/lib.rs

```diff
@@ -1209,3 +1226,200 @@ fn log_time(start: Instant, message: &str) -> Instant {
     }
     now
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Can those be moved in the tests folders? This file is already long enough without tests